pages/1.13/administering-clusters/deploying-a-local-dcos-universe/index.md

```diff
@@ -266,13 +267,13 @@ To deploy a local Universe containing your own set of packages you must build a
 
 1.  Build the `mesosphere/universe` Docker image and compress it to the `local-universe.tar.gz`
 file. Specify a comma-separated list of package names and versions using the `DCOS_PACKAGE_INCLUDE`
-variable. To minimize the container size and download time, you can select only what you need. If you do not use the `DCOS_PACKAGE_INCLUDE` variable, all Certified Universe packages are
-included. To view which packages are Certified, click the **Catalog** tab in the DC/OS web
+variable. To minimize the container size and download time, you can select only what you need. If you do not use the `DCOS_PACKAGE_INCLUDE` variable, all Certified {{ model.packageRepo }} packages are
+included. To view which packages are Certified, click the **{{ model.packageRepo }}** tab in the DC/OS web
```

Comment:
This is always called `Catalog`. It is an actual button in DC/OS UI.

Reply:
OK, I've changed it back to "click the Catalog tab", but left "all Certified {{ model.packageRepo }} packages" as it is.